libs/portal/pages/profile/index.tsx

```
@@ -0,0 +1,18 @@
+  return (
+    <div className="px-2.5 py-5">
+      <Toaster />
+      <Header
+        status={user.profileStatus}
+        firstName={user.details.firstName || ''}
+        lastName={user.details.lastName || ''}
+        avatarUrl={user.details.avatar || ''}
+      />
+      <Questions status={user.profileStatus} user={user.details} />
+    </div>
+  );
```

ESLint

[error] Extra semicolon.

(style/semi)